pretix_mollie/signals.py

```diff
@@ -101,3 +105,20 @@ def register_global_settings(sender, **kwargs):
             ),
         ]
     )
+
+
+@receiver(order_expiry_changed, dispatch_uid="mollie_order_expiry_changed")
+def order_modified(sender: Event, order: Order, **kwargs):
+    payment = order.payments.last()
+    if payment.provider == 'mollie_banktransfer':
```

Comment:
Maybe also check the payment status, otherwise we'll log useless failures for successful or aborted payments?

Reply:
Brainfart on my side when I moved the original check to `update_payment_expiry()` and only added on the Exceptions later on...